src/pytest_cov/plugin.py

```diff
@@ -59,6 +59,9 @@ def pytest_configure(config):
     """Activate coverage plugin if appropriate."""
     if config.getvalue('cov_source'):
         if not config.pluginmanager.hasplugin('_cov'):
+            if not config.option.cov_report:
```

Comment:
Yes, because `pytest_load_initial_conftests` is not run in the xdist process and then coverage would be collected but not reported.